force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -231,48 +231,6 @@
         <shortDescription>UPLOAD_FILES</shortDescription>
         <value>Upload Files</value>
     </labels>
-    <labels>
-        <fullName>ACCOUNT_INSERT_BUTTON</fullName>
-        <language>en_US</language>
-        <protected>true</protected>
-        <shortDescription>Insert Account</shortDescription>
-        <value>Insert Account</value>
-    </labels>
-    <labels>
-        <fullName>NAME_IS_BLANK</fullName>
-        <language>en_US</language>
-        <protected>true</protected>
-        <shortDescription>NAME_IS_BLANK</shortDescription>
-        <value>Name cannot be Blank</value>
-    </labels>
-    <labels>
-        <fullName>PHONE_IS_BLANK</fullName>
-        <language>en_US</language>
-        <protected>true</protected>
-        <shortDescription>Phone Is Blank</shortDescription>
-        <value>Phone cannot be blank</value>
-    </labels>
-    <labels>
-        <fullName>Id</fullName>
-        <language>en_US</language>
-        <protected>true</protected>
-        <shortDescription>Id</shortDescription>
-        <value>Id</value>
-    </labels>
-    <labels>
-        <fullName>Name</fullName>
-        <language>en_US</language>
-        <protected>true</protected>
-        <shortDescription>Name</shortDescription>
-        <value>Name</value>
-    </labels>
-    <labels>
-        <fullName>No_Contacts</fullName>
-        <language>en_US</language>
```

Comment:
why you removing this label?

Reply:
You cannot remove anyone's custom labels

Reply:
Resolved